File: peuler/ex7/ex7.c
#include<stdio.h>
#include<math.h>

int isPrime(unsigned long int n) {
  for (unsigned long int i = 2; i <= (int) sqrt(n); i++)
	if (n % i == 0)
	  return 0;

  return 1;
}

int main() {
  int count = 0;
  unsigned long int number = 2;

  while (count < 10001) {
	if (isPrime(number)) count++;
	number++;
  }

  printf("%lu is the 10001st prime number.\n", number - 1);

  return 0;
}

File: peuler/ex7/ex7.py
import math

def isPrime(number):
    for i in range(2, int(math.sqrt(number) + 1)):
        if (number % i == 0):
            return 0
    return 1

count = 0
number = 2
while (count < 10001):
    if (isPrime(number)):
        count += 1
    number += 1

print(str(number - 1) + " is the 10001st prime number.")
